views/globalSearch/partials/globalSearchList.njk

```diff
@@ -1,6 +1,6 @@
 {% set rows = [] %}
 {% for prisoner in results %}
-  {% set prisonerImage = '<img src="/app/images/' + prisoner.offenderNo + '/data" alt="Image of ' + prisoner.name + '" class="prisoner-search__results-list__image" />' %}
+  {% set prisonerImage = '<img src="/app/images/' + prisoner.offenderNo + '/data" alt="Image of ' + prisoner.name + '" class="prisoner-search__results-list__image" loading="lazy" />' %}
```

Comment:
just added lazy loading to two endpoints at present to try out

Reply:
(which BTW is not supported in Safari but I guess that doesn't matter)